### DISCUSSION TITLE
Restrict Password Rest for Federated Users

### DISCUSSION THREAD

**Discussion by anuragpathak21** [sventorben/keycloak-home-idp-discovery#250] [General]
We have a requirement to restrict password reset for federated users but allow the KC local user (in teh same Realm) to be able to reset password. I was wondering if the Reset Password option comes on password page then it can eb restrcited by this Plugin. As in, the federated user will be redirected to IDP page, hence they will never get an open to rest password.

**Comment by sventorben** [comment]
This extension does not support restricting password reset, sorry.

Regardles on which page the link will be shown, you can assure someone knows the direct link. So I would not bet on this.

**Reply by prsanjay** [reply]
Hi @sventorben Is there any plan to support this? I can handle this by modifying the flow using mappers. But this will not work very first time if user never logged-in to the IdP
